ui/apps/dashboard/src/pages/overview/index.tsx

```diff
@@ -21,7 +21,9 @@ const Overview = () => {
     },
     {
       key: 'karmada-status',
-      label: i18nInstance.t('3fea7ca76cdece641436d7ab0d02ab1b', '状态'),
+      label: capitalize(
```

Comment:
Shall we always use `title case`?

Reply:
+1, for this case `capitalize ` and `titleCase` method share the same effect, but `titleCase` is more suitable.